openstacksdk_plugin/resources/volume/volume.py

```diff
@@ -9,10 +9,12 @@
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
-#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    * See the License for the specific language governing permissions and
-#    * limitations under the License.
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Remove whitespaces between `#` and text. Look to https://github.com/cloudify-incubator/cloudify-utilities-plugin/blob/master/cloudify_cloudinit/__init__.py#L11-L13